.ci/azure-pipelines/documentation.yaml

```diff
@@ -12,16 +12,18 @@ jobs:
     variables:
       BUILD_DIR: '$(Agent.BuildDirectory)/build'
       DOC_DIR: '$(Agent.BuildDirectory)/documentation'
+      CHANGELOG: '$(Agent.TempDirectory)/changelog'
     steps:
       - task: InstallSSHKey@0
         inputs:
           hostName: github.com
           sshPublicKey: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIBh5Yrau/gguTfoNALxhVX77Pgz6y6UWoJRERMKR68ee documentation@pointclouds.org
           sshKeySecureFile: id_ed25519
-      - task: UsePythonVersion@0
-        inputs:
-          versionSpec: '3.6'
-          addToPath: true
+      - script: |
+          $(Build.SourcesDirectory)/.dev/scripts/generate_changelog.py > $CHANGELOG.md
+          grip --export $CHANGELOG.md $CHANGELOG.html
```

Comment:
Technically, you can use pandoc to convert from gfm to html (or html5), no need of grip

Reply:
The advantage of `grip` is that its output looks exactly as if it was rendered by GitHub. And for me that's the goal of this pipeline step, to preview the final changelog (both the content and the look).